target/linux/ramips/dts/mt7628an_qding_qc202.dts

```diff
@@ -0,0 +1,116 @@
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "qding,qc202", "mediatek,mt7628an-soc";
+	model = "Qding QC202";
+
+	leds {
+		compatible = "gpio-leds";
+
+		gpio-relay {
+			label = "gpio-relay";
+			gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
+		};
+
+		ethernet {
+			label = "ethernet";

```

Comment:
There are new ways to declare LEDs now. Take a look at other device tree for the usage of color, function and linux,default-trigger properties.

Reply:
Emmm, this LED has no color. Just simple LED

Reply:
Can u give me some filename?

Reply:
You need to specify what color the LED is (red, green, white or whatever) even if the LED is single-color. Take a look at target/linux/mediatek/dts/mt7981b-openembed-som7981.dts for examples.

Reply:
So simple LED color is white. I think not special also ok. 

I can take a photo for u

![image](https://github.com/user-attachments/assets/4f433d03-0044-41c3-bfce-a60b206c73d4)

Reply:
In fact, this board doesn't have any text on what these LEDs do, so I just defined the names :P

Reply:
So do I need to modify it? I think it's fine now.

Reply:
> So do I need to modify it?

Yup :) You need to change it.

Reply:
like this?

```
		ethernet {
			label = "ethernet";
			gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
			color = <LED_COLOR_ID_WHITE>;
			function = LED_FUNCTION_STATUS;
		};

		wifi {
			label = "wifi";
			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
			color = <LED_COLOR_ID_WHITE>;
			function = LED_FUNCTION_STATUS;
		};
```

Reply:
Yes, the three LEDs are exactly the same except for the different GPIOs

From left to right they are GPIO 55 56 and always on

Reply:
So the rightmost one is the power LED

GPIO 44 and the power LED are active during Uboot

Reply:
There's also a function-enumerator property for differentiating multiple leds of the same function. You can add it to differentiate the two white leds.

Reply:
Can u give me some example or document link? thanks.

Reply:
you need to drop the label property. Manually specifying label in leds is deprecated.

Reply:
> you need to drop the label property. Manually specifying label in leds is deprecated.

Do I need to remove the label of the matrix_keypad LEDs? I didn't find the relevant definition in `dt-bindings/leds/common.h`. This LED is marked as the card swipe area on the original chassis (connected via COM1 cable).